packages/erxes-ui/src/components/richTextEditor/extensions/Table/Cell/Cell.ts

```diff
@@ -7,6 +7,9 @@ export const Cell = TiptapCell.extend({
       style: {
         parseHTML: (element) => element.getAttribute('style'),
       },
+      align: {
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Ensure that the 'align' attribute is properly handled in the table cell extension.